cmd/init_project.go

```diff
@@ -108,15 +108,15 @@ func (o *projectOptions) bindCmdlineFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&o.project.Repo, "repo", "", "name to use for go module, e.g. github.com/user/repo.  "+
 		"defaults to the go package of the current working directory.")
 	cmd.Flags().StringVar(&o.project.Domain, "domain", "my.domain", "domain for groups")
-	cmd.Flags().StringVar(&o.project.Version, "project-version", project.Version2, "project version")
+	cmd.Flags().StringVar(&o.project.Version, "project-version", scaffold.Version2, "project version")
```

Comment:
I do not think that the we should change project for scaffold here. 
IHMO project is a better name since the commands are executed for/in a project and scaffold means the implementations/files scaffolded into the project by default which can be improved/changed by the users.

Reply:
They previously were inside `pkg/scaffold/project` (only the last part is seen in this comment). That package contained mostly `v1` files and some `v2` files. Thus they have been moved inside `pkg/scaffold/files/v1/project` and `pkg/scaffold/files/v2/project` directories. It doesn't make sense to keep `Version1` and `Version2` strings inside `pkg/scaffold/files/v1/project` as that is only for `v1` (same for `pkg/scaffold/files/v2/project`).